manual/source/Tutorial/DataStructures/Alignment/ScoringSchemes.rst

```diff
@@ -0,0 +1,183 @@
+.. sidebar:: ToC
+
+    .. contents::
+
+.. _tutorial-datastructures-alignment-scoringschemes:
+
+Scoring Schemes
+========================
+
+Learning Objective
+  This tutorial introduces you to the scoring systems that can be used in SeqAn to quantify the sequence similarity.
+  You will learn basic techniques to create and modify standard and custom scoring systems capable to satisfy the requirements of a wide range of applications.
+
+Difficulty
+  Basic
+
+Duration
+  0:45h
+
+Prerequisites
+  :ref:`tutorial-getting-started-first-steps-in-seqan`, :ref:`tutorial-datastructures-sequences-alphabets`, :ref:`tutorial-datastructures-sequences`, :ref:`tutorial-datastructures-sequences-string-sets`, :ref:`tutorial-datastructures-sequences-iterators`, :ref:`tutorial-datastructures-alignment-alignment-representation`
+
+The alignment procedures are usually based on the sequences similarity computation described by an alignment scoring system that gives countable information used to determine which sequences are related and which are not. 
+
+Four main biological events must be considered during the sequence alignment: 
+Match when the two compared letters are the same, substitution when a letter is aligned with another, insertion and deletion when in one of the two aligned sequences a letter is aligned with a gap.
+
+.. image:: alig_ins_del.png
+   :width: 500px
+   :align: center
+
+Given an alignment structure that store the two sequences and a scoring scheme, the score of the alignment can be computed as the sum of the scores for aligned character pairs plus the sum of the scores for all gaps.
+
+With refer to the alignment procedure a Scoring Scheme can be defined as the set of rules used to assess the possible biological events that must be considered during the alignment procedure. 
+
+In SeqAn are available several :dox:`Score scoring schemes` to evaluate matches and substitutions, while three different gap models can be applied to consider insertions and deletions events.
+We will first introduce you to the scoring schemes used to evaluate match and substitution. Subsequent, you will learn how to chose the gap model to be implemented in the chosen scoring scheme.
+
+Match/Substitution Evaluation
```

Comment:
We usually speak of "Match/Mismatch" not of "Match/Substitution" in SeqAn, because they are slightly different "match", "mismatch", "(aligned against) gap" represent *states of an alignment position* . These can (and often do) represent *biological processes* like recombination, mutation et cetera and then they represent "conservation", "substitution", "insertion/deletion"; but I still think you shouldn't mix the terms.

The biological terms